### PR TITLE
how-to-attend: add pages based on recent online workshops

### PR DESCRIPTION
**Pull request by rkdarst**
[draft until the sphinx stuff is accepted, then needs TOC linking]

- how-to-attend-online: how to successfully take part online.
- how-to-attend-inperson: same, but for inperson
- Copied from mega-CR and Finland HPC kickstart general information
  and lessons learned.
- This is not designed to replace installation instructions, but more
  general mental and physical preparation.  Should this me moved to
  installation instructions sometime?
- To review: deserves a detailed reading to make sure things are what
  we agree on, typographical errors, and to ensure it general positive
  message and doesn't demotivate people.

### CODE REVIEW

**Comment by bast** [comment]
Should I take care of the TOC part and finishing up this PR?

**Comment by rkdarst** [comment]
updated and ready to go

File: how-to-attend-online.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+
+
+## Time management
+
+Despite what most people think, attending things online can be harder
+than in-person.
+
+Don't schedule overlapping meetings, reserve the entire timeslots,
+minimze distractions.  It's easy to think you can do multiple things
```
Comment (wikfeldt) [inline how-to-attend-online.md:100]:
```suggestion
minimize distractions.  It's easy to think you can do multiple things
```

File: how-to-attend-inperson.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+Usually, you'll need to look at both the lesson webpage and the window
+where you are doing the exercises at the same time.  Consider how you
+can arrange windows to do this best.
+
+
+
+## Time management
+
+Try your very best to attend the whole workshops; at least don't miss
```
Comment (wikfeldt) [inline how-to-attend-inperson.md:76]:
```suggestion
Try your very best to attend the whole workshop; at least don't miss
```

File: how-to-attend-online.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+
+
+## Social
+
+Attend with someone!  Register together and try to be in their same
+group.  This will create a network of learning and practice that will
+last much longer.
+
+If you can attend a group, that is even better.  You can bring your
```
Comment (wikfeldt) [inline how-to-attend-online.md:60]:
```suggestion
If you can attend as a group, that is even better.  You can bring your
```

File: how-to-attend-inperson.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+
+
+## Social
+
+Attend with someone!  Register together and try to sit near them.
+This will create a network of learning and practice that will last
+much longer.
+
+If you can attend a group, that is even better.  You can bring your
```
Comment (wikfeldt) [inline how-to-attend-inperson.md:55]:
```suggestion
If you can attend as a group, that is even better.  You can bring your
```

File: how-to-attend-online.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+quiet and listen all the time.  Try to be in a place where you can
+speak without disturbing others.  By the same token, you'll be
+listening for a long time, and your ears may get tired of headphones.
+If you have good enough external speakers, be somewhere that you can
+use them (perhaps only sometimes - when it doesn't interfere with your
+microphone.)
+
+If you work in a large office, consider attending from home or in a
+meeting room so that you can speak and listen more freely.  If you
+need and extra monitor or more comfortable seating space and don't
```
Comment (wikfeldt) [inline how-to-attend-online.md:88]:
```suggestion
need an extra monitor or more comfortable seating space and don't
```

**Pull request merged**